src/meta/metainfo.go

```diff
@@ -85,6 +85,7 @@ const (
 // For example, \array_slice($arr) returns type of element (OverrideElementType) of the ArgNum=0
 type FuncInfoOverride struct {
 	OverrideType OverrideType
+	WithNull     bool // whenever we need to add null type (usually for OverrideClassType)
```

Comment:
How about a new override type, OverrideNullableClassType.
You can share the code by moving the common logic to the function and add null type to the OverrideNullableClassType case.

Reply:
I thought about it, maybe this is really a better solution.

Reply:
Done.